cpp/src/slice2swift/SwiftUtil.cpp

```diff
@@ -257,16 +255,6 @@ Slice::getTopLevelModule(const ContainedPtr& cont)
     return m;
 }
 
-ModulePtr
-Slice::getTopLevelModule(const TypePtr& type)
-{
-    // TODO this check is redundant. `InterfaceDecl` _is_ a `Contained`. This whole function is weird though.
```

Comment:
The original code in 3.7 made sense. 

https://github.com/zeroc-ice/ice/blob/d53f0a955fd1e925fd0f6159953d781b09b9b1ba/cpp/src/slice2swift/SwiftUtil.cpp#L213-L221